deployments/compose/README.md

```diff
@@ -1,21 +1,31 @@
 # lakeFS `docker-compose`
 
 Compose file for lakeFS using variety of services lakeFS can work with.
+Automatically creates a minIO bucket and lakeFS repository as part of the start up.
 
+To spin up the environment, run:
+```sh
+docker compose up -d
+```
+
+That's it! To check the status of the containers:
+```sh
+docker compose ps
+```
 
 ### Hive Client
 
 Can access Hive Server using client, run under 'client' profile:
 
 ```sh
-docker-compose --profile client run --rm hive-client
+docker compose --profile client run --rm hive-client
```

Comment:
I remember Barak had some bad experience with `docker compose`
Not sure this was the case. @nopcoder is it OK to use `docker compose` here?